setup.py

```diff
@@ -32,8 +32,10 @@
 
```

Comment:
Please add this file in another PR. One PR should fix only one small issue :)

Reply:
This is the dependency for beautifulsoup and lxml libraries used in this PR. Is it supposed to be in a different PR?

Reply:
Ideally this whole html thing should be a different PR but in the interest of time, I'd approve it.